client/src/templates/Challenges/rechallenge/transformers.js

```diff
@@ -74,27 +75,21 @@ export const babelTransformer = cond([
   [stubTrue, identity]
 ]);
 
+const sass = new WorkerExecutor('sass');
```

Comment:
Just naming here. I think `sassWorker` would be better, just so I do not confuse with `Sass` 3 months later 😄

Reply:
have renamed